src/test_driver/nrfconnect/main/runner.cpp

```diff
@@ -29,13 +30,24 @@ using namespace ::chip::DeviceLayer;
 
 LOG_MODULE_REGISTER(runner, CONFIG_MATTER_LOG_LEVEL);
 
-extern "C" int main(void)
+extern "C" {
+void console_init(void) {}
```

Comment:
This is a hack that shadows Zephyr function definitions. Could you at least add a TODO comment that the tests should rather configure and use Zephyr's console subsystem?

Reply:
I've checked, and this hack was necessary in the first version of my changes. This is no longer required, so I removed it.

Reply:
ok, thanks! :)